_docs_use-cases/use-case-consumption-delete-a-third-party-owned-relationshipattribute-and-notify-peer.md

```diff
@@ -36,3 +36,26 @@ require:
 required_by:
 # End automatic generation
 ---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case allows you to delete a third party owned RelationshipAttribute, i.e. a [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) that exists in a Relationship with a peer, but is not owned by you or this peer but by a third party.
+
+## Parameters
+
+- The `attributeId` of the third party owned RelationshipAttribute you want to delete.
+
+## On Success
+
+- The third party owned RelationshipAttribute will be deleted.
+- All predecessors of the third party owned RelationshipAttribute will be deleted.
+- If the third party owned RelationshipAttribute was succeeded, the `succeeds` property of the successor will be set to undefined.
+- A [Notification]({%link _docs_integrate/data-model-overview.md %}#notification) with [ThirdPartyOwnedRelationshipAttributeDeletedByPeerNotificationItem]({%link _docs_integrate/data-model-overview.md %}#thirdpartyownedrelationshipattributedeletedbypeernotificationitem) is sent to the peer of the Attribute, informing them that you deleted the Attribute. Technically, the Attribute of the owner and all predecessors will get a `deletionInfo` with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
```

Comment:
The sentence "Technically, the Attribute of the owner and all predecessors will get a `deletionInfo` with `deletionStatus` `"DeletedByPeer"`..." must be reformulated, as its not the Attribute of the owner, but the corresponding third party owned RelationshipAttribute of the peer, we are talking about here.